File: sistema_facturacion_api/sistema_facturacion_api.Data/DTOs/TblDetalleDeFacturasDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace sistema_facturacion_api.Data.DTOs
{
    public class TblDetalleDeFacturasDTO
    {
        public int? Id { get; set; }
        public int? FacturaId { get; set; }
        public int? ProductoId { get; set; }
        public string? NombreProducto { get; set; }
        public decimal? PrecioUnidad { get; set; }
        public int? CantidadProducto { get; set; }
        public DateTime? FechaDeRegistro { get; set; }
    }
}

File: sistema-facturacion-front/src/app/models/dtos/tbl-detalle-de-facturas-dto.ts

export class TblDetalleDeFacturasDTO{

  // id: number;
  facturaId: number;
  productoId: number;
  nombreProducto: string;
  precioUnidad: number;
  subTotal: number;
  descuento: number;
  iva: number;
  totalPagado: number;
  formaDePagoId: number;
  cantidadProducto: number;
  // fechaDeRegistro: Date;

  constructor(){
    // this.id = 0;
    this.facturaId = 0;
    this.productoId = 0;
    this.nombreProducto = '';
    this.precioUnidad = 0;
    this.subTotal = 0;
    this.descuento = 0;
    this.iva = 0;
    this.totalPagado = 0;
    this.formaDePagoId = 0;
    this.cantidadProducto = 0;
    // this.fechaDeRegistro = new Date();
  }
}
